### DISCUSSION TITLE
SrcAddr and DstAddr fails past 15 days?

### DISCUSSION THREAD

**Discussion by LoveSkylark** [akvorado/akvorado#1028] [Q&A]
I am currently exploring Akvorado, and I it appears quite promising. However, I have encountered a limitation where I am unable to access data beyond a 15-day timeframe when attempting to retrieve source or destination addresses.  The "Apply" button fails to initiate.

Is this limitation intentional?, or if it is possibly due to a buffer capacity?, or have I misconfigured my server somehow?
Could this issue be resolved through a setting adjustment, or is there another underlying cause that I should consider?

**Comment by vincentbernat** [comment]
Yes, it's intentional. See the `resolution` setting in the documentation: https://demo.akvorado.net/docs/configuration#clickhouse
